src/main/java/apoc/load/util/LoadJdbcConfig.java

```diff
@@ -13,6 +15,8 @@
 
     private ZoneId zoneId = null;
 
+    private Map<String,String> credentials;
```

Comment:
I would make this explicit into username and password.
And add a method "hasCredentials"